storage/jellyfish-merkle/src/lib.rs

```diff
@@ -196,9 +202,54 @@ pub struct StaleNodeIndex {
 /// which is a vector of `hashed_account_address` and `new_value` pairs.
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
 pub struct TreeUpdateBatch<K> {
-    pub node_batch: NodeBatch<K>,
-    pub stale_node_index_batch: StaleNodeIndexBatch,
-    pub node_stats: Vec<NodeStats>,
+    pub node_batch: Vec<Vec<(NodeKey, Node<K>)>>,
```

Comment:
does this need to be vector of vector?

Reply:
this is to reduce the cost of `memcpy`s that happens when I "combine" the branches.